concrete_datastore/api/v1/authentication.py

```diff
@@ -104,7 +104,7 @@ class URLTokenExpiryAuthentication(TokenExpiryAuthentication):
 
     def authenticate(self, request):
         token = request.GET.get('c_auth_with_token', '')
-        if token == '':
+        if token == '':  # nosec
```

Comment:
Otherwhise, replace by

```python
if not token:
    return
```

this avoid the "# nosec" usage

Reply:
PR updated